src/mono/wasm/wasm.proj

```diff
@@ -158,8 +158,10 @@
     <PropertyGroup>
       <_EmccExportedRuntimeMethods>&quot;[@(EmccExportedRuntimeMethod -> '%27%(Identity)%27', ',')]&quot;</_EmccExportedRuntimeMethods>
       <_EmccExportedFunctions>@(EmccExportedFunction -> '%(Identity)',',')</_EmccExportedFunctions>
+      <EmccInitialHeapSize>536870912</EmccInitialHeapSize>
```

Comment:
cc @lewing - this changes the initial size that users were getting with no relinking/aot, from 16 -> 512 .

Reply:
I'm just wondering whether this would break any one, like running on mobile browsers?

Reply:
Yes, I think 512 is too high for mobile, especially because android chrome is still 32-bit.